content/docs/how-we-work/coding-principles-standards.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Coding Principles and Standards
+weight: 4
+description: At Axelerant, we write code not only to solve a specific problem but also to provide a reusable solution that can be used in a community level. We have principles that can guide us to the desired code standards.
+---
+
+# Coding Principles and Standards
+
+At Axelerant, we write code not only to solve a specific problem but also to provide a reusable solution that can be used in a community level. We have principles that can guide us to the desired code standards.
+
+## Coding Principles
+
+- We use the correct layer of abstraction.
+
+- We write clean, simple code and explain our purpose with comments.
+
+- We contribute back everything we can, because it supports us, our clients and the community.
+
+- We manage patches in a standard way so that we can know if code/contrib has been changed, how and why. Every change should be contributed back, and the patch name should include the drupal.org issue NID.
+
+- We write secure code.
+
+- We write code that anticipates change, so that we can implement (most) changes in an easy and Agile way.
+
+- We write API layer that doesn't assume a particular UI, so that the API could be used in multiple (and unpredictable) situations (For example, used in these UIs: Web site, drush, rest, SMS).
```

Comment:
```suggestion
- We write API layer that doesn't assume a particular UI, so that the API could be used in multiple (and unpredictable) situations (For example, used in these UIs: Web site, drush, REST, SMS).
```